hnginternship5/lucid
# Lucid test

### TITLE OF PR
feature clean up and bug fix
### PURPOSE OF PR
bug on save to draft fixed

### FEATURES IMPLEMENTED
swal and php control statement

### HOW TO TEST
for the about page you can't test in full now
but when logged in as the owner you will see an edit button
for draft create a draft and get the success pop up
